hazelcast/src/main/java/com/hazelcast/nio/ascii/TextReadHandler.java

```diff
@@ -210,6 +210,9 @@ private boolean isCommandTypeEnabled(TextCommand command) {
                 boolean isMancenterRequest = uri.startsWith(HttpCommandProcessor.URI_MANCENTER_CHANGE_URL);
                 boolean isClusterManagementRequest = uri.startsWith(HttpCommandProcessor.URI_CLUSTER_MANAGEMENT_BASE_URL);
                 boolean isHealthCheck = healthcheckEnabled && uri.startsWith(HttpCommandProcessor.URI_HEALTH_URL);
```

Comment:
I would recommend handling the BAD_REQUEST inside the Processor see. `HttpGetCommandProcessor` rather than in the TextReadHandler; which is a very abstract protocol input handler. I know we already have some form of validation here, but I recon its better to keep it from getting more populated. Also, I would just change **there**, the `.startsWith` with `.equals` and let the default 400 propagate.